images/rabbitmq/configs/latest.apko.yaml

```diff
@@ -7,8 +7,18 @@ contents:
     - rabbitmq-server
     - wolfi-baselayout
     - bash
+    - glibc-locale-en
+    - tzdata
+    # - procps do we need this?
+    # - su-exec do we need this?
 
 paths:
+  - path: /var/lib/rabbitmq
```

Comment:
I added this because I was getting the following error when testing locally

```
docker run --platform linux/aarch64  -p 5672:5672 --rm rabbit:test
2023-01-25 09:37:56.558129+00:00 [error] <0.146.0> Failed to create cookie file '/var/lib/rabbitmq/.erlang.cookie': eacces
2023-01-25 09:37:56.571452+00:00 [error] <0.144.0>     supervisor: {local,net_sup}
2023-01-25 09:37:56.571452+00:00 [error] <0.144.0>     errorContext: start_error
2023-01-25 09:37:56.571452+00:00 [error] <0.144.0>     reason: {"Failed to create cookie file '/var/lib/rabbitmq/.erlang.cookie': eacces",
```